common/util.go

```diff
@@ -565,3 +566,12 @@ func CalculateAudioDuration(audio types.File) (int64, error) {
 
 	return duration, nil
 }
+
+func ExtensionByType(contentType string) (string, error) {
```

Comment:
Yes, or should i switch to the go-tools repo version of this? Just found it existed there yesterday.

Reply:
I added a test for `image/png` which is the only used extension for this.  Future will look at runner returning multipart/form-data response so can have actual file included.

Note `video/mp4 returns` as .f4v.

Reply:
Closing out this conversation. The ExtensionByType function was updated to set extensions directly using a switch block instead of the `mime` package because the `mime` package returns multiple options for `video/mp4`.